doc/index.rst

```diff
@@ -103,16 +109,18 @@ To start using CI config stored in this repo
 
 With the following few short steps, you can start in your client repository using CI confiurations stored in here (`industrial_ci` repository).
 
-1. Don't forget to activate Travis CI on your github repository (you may do so on https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or https://travis-ci.org/profile/YOUR_GITHUB_USER).
+1. Don't forget to activate CI for your repository.
+ - For Travis CI and GitHub, you may do so on https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or https://travis-ci.org/profile/YOUR_GITHUB_USER).
 
-2. In `.travis.yml` file in your client repo, add in `before_config` section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .ci_config`, like below:
+2. In `CI config <#terminology>`_ file in your client repo, add in `before_config` section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .ci_config`, like below:
 
 ::
 
   before_config:
     - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
   script:
-    - .ci_config/travis.sh
+    - .ci_config/ci_main.sh  (Travis CI with GitHub)
```

Comment:
Still ` .ci_config/travis.sh`

Reply:
There is even the generic `.ci_config/ci.sh` script that can be used with other CI systems or custom solutions like Jenkins.